utils/make_sweep_dir.py

```diff
@@ -0,0 +1,89 @@
+""" Run ther create directory methods of the enseble_tools module
+"""
+
+from ensemble_tools import create_dir_tree, create_dict_sweep
+
+import argparse
+import sys
+
+
+def main():
+    # Make the argument parser
+    parser = argparse.ArgumentParser(
+        prog="makeSweepDir", description="Makes a sweep directory for FabNeso"
```

Comment:
```suggestion
        description="Makes a sweep directory for FabNeso"
```

Better to leave `prog` as the default (which will correspond to name of script being called) to avoid this getting out of sync with filename as this has.